### PR TITLE
add sigmoid custom grad for prim

### PR DESCRIPTION
**Pull request by veyron95**
### PR types

Others
### PR changes

Others
### Describe

Pcard-66975
add sigmoid custom grad for prim

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/prim/prim/vjp/eager/test_comp_eager_sigmoid_grad.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/prim/prim/vjp/eager/test_comp_eager_sigmoid_grad.py:1]: Make sure sigmoid tested in op test

**Review by JiabinYang — COMMENTED** [review]
one comment

File: python/paddle/fluid/tests/unittests/prim/prim/vjp/eager/test_comp_eager_sigmoid_grad.py (hunk 1 above)
Comment (veyron95) [reply python/paddle/fluid/tests/unittests/prim/prim/vjp/eager/test_comp_eager_sigmoid_grad.py:1]: yes, sigmoid was tested in op test

**Review by JiabinYang — APPROVED** [review]
LGTM

**Pull request merged**